src/main/java/Duke.java

```diff
@@ -1,10 +1,166 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+
+
 public class Duke {
-    public static void main(String[] args) {
+    public static ArrayList<Task> manager = new ArrayList<>();
+
+    public static void main(String[] args) throws IOException {
+        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        greet();
+        addLineBreak();
+
+        while (true) {
+            String instruct = br.readLine();
+            try {
+                if (instruct.equals("bye")) {
+                    endSession();
+                    break;
+                } else if (instruct.equals("list")) {
+                    reportList();
+                } else {
+                    String[] details = instruct.split(" ");
```

Comment:
Maybe it will be better if you could break up this chunk of code into different methods? For example, having a method that will take in a `String userInput` as a parameter and then execute the corresponding command.